src/common/engine.ts

```diff
@@ -974,7 +974,7 @@ export class CurrencyEngine<
     checkEdgeSpendInfo(edgeSpendInfo)
 
     for (const st of edgeSpendInfo.spendTargets) {
-      if (st.publicAddress === this.walletLocalData.publicKey) {
+      if (!skipChecks && st.publicAddress === this.walletLocalData.publicKey) {
```

Comment:
This affects a lot of coins. However, do I see how it does go along with the variable name. I guess it's fine?

Reply:
I thought about adding a `allowSpendToSelf` but I kind of like `skipChecks` as it should mean override the common checking and allow the engine to handle it

Reply:
Sounds good.